### DISCUSSION TITLE
GET dev.json request

### DISCUSSION THREAD

**Discussion by jrean** [nuxt/nuxt#25882] [Questions] [answered]
On a very basic nuxt 3 install (currently 3.9.3), a simple page load will send a `_nxtsb/builds/meta/dev.json` GET request as well as a periodic `/builds/latest.json` GET request! What is that? How can I disable it?

```
export default defineNuxtConfig({
  devtools: { enabled: false },
  ssr: false,
  srcDir: './src/',
...
```

**Comment by TheAlexLichter** [comment, marked as answer]
This is the [appManifest](https://github.com/nuxt/nuxt/pull/23448), containing information about the application, such as routes and a build identifier. It is [enabled by default](https://github.com/nuxt/nuxt/pull/23448).
You can disable it via `experimental.appManifest` in the config ;)

**Reply by jrean** [reply]
@manniL Thank you!
